### DISCUSSION TITLE
Is live reload possible with full-stack workspace?

### DISCUSSION THREAD

**Discussion by Tungsten78** [nx-dotnet/nx-dotnet#606] [General]
I have a nx workspace with angular app and nx-dotnet webapi all wired through the generated swagger client (typescript-angular).
* Each web app and webapi reload themselves nicely for code changes within their respective projects.

Goal:
Change in controller implementation propagates to angular client via a live reload.

Is this possible out the box with nx?   Or does this require special configuration on the angular side in webpack dev-server perhaps?

**Comment by AgentEnder** [comment]
It is possible, you would probably want to run something like this in a second terminal to watch the backend and generate types

`nx watch --projects webapi  --includeDependencies -- nx codegen generated-webapi-swagger`

**Reply by AgentEnder** [reply]
We need to write up a recipe / guide for the codegen stuff and it could include this too... if you are up for a docs contribution that would be an **_excellent_** one

**Reply by Tungsten78** [reply]
With the watch, codegen fires as expected.   Though now faced with the web project's watch triggering before codegen is done resulting in a build error - module cannot be found.  For some reason, this seems to put web pack into a funky state where subsequent watch changes aren't fixing the build either.
